corehq/apps/es/management/commands/elastic_sync_multiplexed.py

```diff
@@ -32,31 +36,65 @@ def start_reindex(self, cname):
             raise IndexNotMultiplexedException("""Index not multiplexed!
             Sync can only be run on multiplexed indices""")
 
-        self.source, self.dest = self._get_source_destination_indexes(adapter)
-
-        task_id = es_manager.reindex(self.source, self.dest)
-
+        source_index, destination_index = self._get_source_destination_indexes(adapter)
+
+        logger.info(f"Preparing index {destination_index} for reindex")
+        self._prepare_index_for_reindex(destination_index)
+
+        logger.info("Starting ReIndex process")
+        task_info = es_manager.reindex(source_index, destination_index)
+        logger.info(f"Copying docs from index {source_index} to index {destination_index}")
+        task_id = task_info.split(':')[1]
+        print("\n\n\n")
+        logger.info("-----------------IMPORTANT-----------------")
+        logger.info(f"TASK ID - {task_id}")
+        logger.info("-------------------------------------------")
+        logger.info("Save this Task Id, You will need it later for verifying your reindex process")
+        print("\n\n\n")
         # This would display progress untill reindex process is completed
-        check_task_progress(task_id)
+        check_task_progress(task_info)
 
+        print("\n\n")
         self.perform_cleanup(adapter)
 
+        logger.info("Preparing Index for normal use")
+        self._prepare_index_for_normal_usage(adapter.secondary)
+        print("\n\n")
+
+        self._get_source_destination_doc_count(adapter)
+
+        logger.info(f"Verify this reindex process from elasticsearch logs using task id - {task_id}")
+        print("\n\n")
+        logger.info("You can use commcare-cloud to extract reindex logs from cluster")
+        print("\n\t"
+            + "cchq <env> run-shell-command elasticsearch \"cat /opt/data/elasticsearch*/logs/*es.log"
+            + f" | grep '{task_id}.*ReindexResponse'\""
```

Comment:
No need to abuse the cat :slightly_smiling_face: 
```suggestion
            + "cchq <env> run-shell-command elasticsearch \""
            + f"grep '{task_id}.*ReindexResponse' /opt/data/elasticsearch*/logs/*es.log\""
```